doc/alarm.md

```diff
@@ -184,17 +188,30 @@ public interface SmsSender {
 
 **C. Webhook alarm service**
 
-Webhook alarm service is the api of Pinpoint alarm over HTTP. To receive alarms you will need to prepare the receiver server. There is an example of webhook slack receiver project, see 3. Others - [5) An example of webhook receiver project](##3.Others). The payloads of the webhook have the different schemas depending on the alarm checker type. you can see the example of the payload schemas in 3.Others - [6) The Specification of webhook payloads and the examples](##3.Others). If you have an previous release of pinpoint-2.1.1, you should change the schema of alarm_rule table.
+Webhook alarm service is the API of Pinpoint alarm over HTTP. 
```

Comment:
I think 
'Webhook alarm service is a feature that can transmit Pinpoint's Alarm message through Webhook API. '
looks better. what do you think?

Reply:
yeah I think so

Reply:
why is this duplicated with the sentence below?

Reply:
Thanks, I will fix it